core/server/common/src/main/java/alluxio/master/journal/raft/RaftJournalSystem.java

```diff
@@ -375,14 +376,32 @@ private synchronized void initServer() throws IOException {
     // build server
     mServer = RaftServer.newBuilder()
         .setServerId(mPeerId)
-        .setGroup(mRaftGroup)
+        .setGroup(mInitialRaftGroup)
         .setStateMachine(mStateMachine)
         .setProperties(properties)
         .setParameters(parameters)
         .build();
   }
 
-  private RaftClient createClient() {
+  /**
+   * @return current raft group
+   */
+  @VisibleForTesting
+  public synchronized RaftGroup getCurrentGroup() {
```

Comment:
Is there any callback hook or some ways we can get the current group automatically instead of checking for each gainPrimacy or checkpoint?